pkg/cableengine/healthchecker/healthchecker.go

```diff
@@ -172,9 +183,12 @@ func (h *controller) endpointDeleted(obj runtime.Object, _ int) bool {
 	h.Lock()
 	defer h.Unlock()
 
-	if pingerObject, found := h.pingers[endpointDeleted.Spec.CableName]; found {
-		pingerObject.Stop()
-		delete(h.pingers, endpointDeleted.Spec.CableName)
+	for _, family := range h.config.SupportedIPFamilies {
+		cableName := endpointDeleted.Spec.GetFamilyCableName(family)
+		if pingerObject, found := h.pingers[cableName]; found {
```

Comment:
We should be persistent with field name represents endpoint family cable name , we use 'cableName' here and 'endpointName'  in startPinger